src/components/Dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC, HTMLProps, ReactElement } from "react";
+
+export type DropdownProps = {
+  position?:
+    | "dropdown-top"
+    | "dropdown-bottom"
+    | "dropdown-left"
+    | "dropdown-right";
+  align?: "dropdown-end";
+  renderButton: ReactElement;
+  items: (HTMLProps<HTMLLIElement> & {
+    "data-testid"?: string;
+    onClick?: () => void;
+    renderItem: ReactElement;
```

Comment:
Sorry I was wrong here, you can remove string only if you replace `ReactElement` with `ReactNode`.

With `ReactNode` you don't need to wrap in `<span>` the strings.